src/babashka/fs.cljc

```diff
@@ -55,7 +55,9 @@
   (^Path [f]
    (as-path f))
   (^Path [parent child]
-   (as-path (io/file (as-file parent) (as-file child))))
+   (if parent
```

Comment:
There's a number of tests calling path with `parent` being `nil` and this fixes these failures.